e2e/deployment.go

```diff
@@ -0,0 +1,106 @@
+package e2e
```

Comment:
please add the license and copyright header

Reply:
I guess all the files in e2e/*.go miss the copyright info, I thought there might be some reason for this :-)
I will fix this up, thanks!

Reply:
Yeah, they are missing in lots of files. Adding the license afterwards is bad practice... So, we'll try to add the license when new files get created.

Reply:
Ah! I almost made the changes locally and made a commit to fixing the license in all the files missing them ;-)
I will drop if that turns out to be a bad practice.